src/NET/MvvmDialogs/DialogTypeLocators/NamingConventionDialogTypeLocator.cs

```diff
@@ -50,7 +57,12 @@ public Type Locate(INotifyPropertyChanged viewModel)
 
         private static string GetDialogName(Type viewModelType)
         {
-            string dialogName = viewModelType.FullName.Replace(".ViewModels.", ".Views.");
+            string dialogName;
```

Comment:
Instead of if-statements, can we write this code like:

```
string dialogName = viewModelType.FullName
    .Replace(".ViewModel.", ".View.")
    .Replace(".ViewModels.", ".Views.");
```

Your code will probably not work since most view models are suffixed with `ViewModel`, and that will run the first if-statement, which possibly is the reason to why the [tests](https://ci.appveyor.com/project/FantasticFiasco/mvvm-dialogs) now fail.